inst/htmljs/animint.js

```diff
@@ -1549,8 +1604,8 @@ var animint = function (to_select, json_file) {
             return scales.y(d["middle"]);
           })
           .style("stroke-dasharray", get_dasharray)
-          .style("stroke-width", get_size)
-          .style("stroke", get_colour);
+          .style("stroke-width", get_size);
+          select_style_fun(g_info, e);
```

Comment:
Hi @Faye-yufan I see that `.style("stroke", get_colour)` has been replaced by `select_style_fun(g_info, e)` which I guess works but is a bit confusing to me (previously it was clear that `get_colour` was responsible for stroke, but now it is not clear that `select_style_fun` is responsible for stroke -- does it also handle the alpha/opacity?). Did you trying modifying the `get_colour` function and that did not work for some reason?

Reply:
yes, `select_style_fun` is responsible for both stroke and opacity. Since I'm still following the `ifSelectedElse` logic, which has to update selectors by get_colour/ get_colour_off two separate functions, I haven't tried modifying `get_colour` or getting rid of `ifSelectedElse`.

Reply:
Will try modifying the get_colour function.